src/bastion/azext_bastion/_validators.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import ipaddress
+from azure.cli.core.azclierror import InvalidArgumentValueError
+
+
+def validate_ip_address(namespace):
```

Comment:
Nit: Do we need to raise validation error when target_ip_address is None or empty?

Reply:
![image](https://user-images.githubusercontent.com/56377848/214398038-946e8b07-19b3-47a0-ba62-d40673c898ba.png)